frontend/institution/management_institution_page.html

```diff
@@ -1,6 +1,6 @@
 <md-content layout="row" md-colors="{background: 'grey-200'}" layout-align="center">
   <!-- CONTENT -->
-  <div flex-lg="90" flex-gt-lg="70" layout="row" layout-align="space-between"
+  <div flex-lg="90" flex-gt-lg="70" flex-md="100" layout="row" layout-align="space-between"
```

Comment:
It's missing the flex default, to guarantee the div in the rage outside the definition being responsive.